test/ci.sh

```diff
@@ -72,13 +78,13 @@ main() {
     linux_steps
   fi
 
+  $maybe_sudo rm -rf /usr/share/ramalama /opt/homebrew/share/ramalama /usr/local/share/ramalama
```

Comment:
The problem with this is it modifies the host system when users run `make test`
 If we want to test this, we should just do it in the ci/cd system not as part of the `make test` job.

Reply:
We have to at least run:

mac_steps -> ./install.py

or some equivalent... To install the python dependancies for macOS...